prov/gni/include/gnix_datagram.h

```diff
@@ -118,6 +119,62 @@ enum gnix_dgram_poll_type {
 	GNIX_DGRAM_BLOCK
 };
 
+/**
+ * @brief GNI datagram structure
+ *
+ * @var list                 list element for manging datagrams in llists
```

Comment:
manging => managing (unless you're taking about a skin disease here, in which case I need a better explanation)